wf_pages/models.py

```diff
@@ -134,7 +135,7 @@ class Meta:
         verbose_name_plural = "Pages"
 
 
-class MollyWingateBlogPage(WfPage):
+class MollyWingateBlogPage(DrupalFields, WfPage):
```

Comment:
We don't need to inherit from `DrupalFields` here, since `WfPage` already inherits from `DrupalFields`. See if that fixes the method resolution order error.
```suggestion
class MollyWingateBlogPage(WfPage):
```

Reply:
I tried this, but it still returns an error:
`django.db.utils.ProgrammingError: column "drupal_path" of relation "community_onlineworship" already exists`

Reply:
I was able to run the migrations successfully in your branch. Try running `python manage.py reset_db` and then re-running the `python manage.py migrate` command.

Reply:
Yes! Same here.
Thanks for your patience!